baseline/sweep

```diff
@@ -42,41 +54,92 @@ train() {
 
 ensemble() {
     for LEVEL in low medium high; do
-        # TODO(kbg): Add test ensembles when test data becomes available.
-        for DEV in "${DATA}/${LEVEL}/"*"_dev.tsv"; do
-            LG="$(basename ${DEV} _dev.tsv)"
-            python trans/ensembling.py \
-                --gold="${DATA}/${LEVEL}/${LG}_dev.tsv" \
-                --systems "${OUTPUT}/${LEVEL}/${LG}/"*"/dev_beam${BEAM_WIDTH}.predictions" \
-                --output="${OUTPUT}/${LEVEL}/${LG}/ensemble"
+        for TRAIN in "${DATA}/${LEVEL}/"*"_train.tsv"; do
+            LG="$(basename ${TRAIN} _train.tsv)"
+            if [ ${LANGS} ] && ! [[ " ${LANGS} " =~ " ${LG} " ]]
```

Comment:
Maybe `[[ ${LANGS} ]]` for consistency?

Reply:
Think that should work. I'm still fuzzy on test brackets though.

Reply:
I just pick one of the two styles. If you're using [[ anywhere I'd use it everywhere. [[ is better anyways.